src/main/webapp/styles/style.css

```diff
@@ -261,3 +260,28 @@ body {
     transition: font-size 0.3s ease-in-out;
   }
 }
+
+.loader {
+  animation: spin 2s linear infinite;
+  border: 16px solid #f3f3f3;
```

Comment:
The `border` attribute seems redundant, given that you set them individually for each of the four directions.